internal/runners/projects/projects_test.go

```diff
@@ -23,14 +31,14 @@ func TestProjects(t *testing.T) {
 	httpmock.Register("GET", "/organizations/organizationName/projects")
 
 	catcher := outputhelper.NewCatcher()
-	pjs := NewProjects(catcher.Outputer, authentication.Get())
+	pjs := NewProjects(catcher.Outputer, authentication.Get(), &configMock{})
 
 	projects, fail := pjs.fetchProjects()
 	assert.NoError(t, fail.ToError(), "Fetched projects")
 	assert.Equal(t, 1, len(projects), "One project fetched")
-	assert.Equal(t, "test project", projects[0].Name)
+	assert.Equal(t, "test project (test description)", projects[0].Name)
 	assert.Equal(t, "organizationName", projects[0].Organization)
-	assert.Equal(t, "test description", projects[0].Description)
+	assert.Equal(t, []string{"/some/local/path/"}, projects[0].LocalCheckouts)
```

Comment:
Attempted to write integration tests for this however due to the string wrapping of the table output the test suite is not able to match the output easily.